packages/api/src/webhooks/cannon.ts

```diff
@@ -23,30 +24,37 @@ export default class WebhookCannon {
   running: boolean;
   verifyUrls: boolean;
   resolver: any;
-  constructor({ db, store, verifyUrls }) {
+  queue: MessageQueue;
+  constructor({ db, store, verifyUrls, queue }) {
     this.db = db;
     this.store = store;
     this.running = true;
     this.verifyUrls = verifyUrls;
     this.resolver = new Resolver();
+    this.queue = queue;
     // this.start();
   }
 
   async start() {
     console.log("WEBHOOK CANNON STARTED");
-    if (this.running) {
-      await this.db.queue.setMsgHandler(this.processEvent.bind(this));
-    }
+    await this.queue.consume(this.handleQueueMsg.bind(this));
   }
 
-  async processEvent(msg: Notification) {
-    console.log("EVENT TRIGGERED ON THE WEBHOOK");
-    let event = await this.db.queue.pop(this.onTrigger.bind(this));
-    console.log("event: ", event);
+  async handleQueueMsg(data: any) {
+    let message = JSON.parse(data.content.toString());
+    console.log("webhookCannon: got message", message);
+    await this.onTrigger(message);
+    this.queue.ack(data);
```

Comment:
Should we `nack` in case there is some error with the `onTrigger` callback? Or is there an automatic timeout for that or something?